util/envDynawoAlgorithms.sh

```diff
@@ -686,13 +687,28 @@ elif [ -n "$ZSH_VERSION" ]; then
 fi
 
 ## force build_type for specific cases
-LAUNCH_COMMAND=$*
 MODE=$1
+ARGS=""
+while [[ $# -gt 0 ]]
```

Comment:
In execDynAglo we do while (($#)); do, better harmonize it

Reply:
OK